lib/src/single_line_comments/single_line_comment.dart

```diff
@@ -23,6 +23,8 @@ class SingleLineComment {
 
   late final isReadonly = _checkIfReadonly();
 
+  int get lastCharacterIndex => characterIndex + outerContent.length;
```

Comment:
Remove?

Reply:
If we keep it, it is `endCharacterIndex`

Reply:
removed